sunpy/util/tests/test_util.py

```diff
@@ -62,3 +64,42 @@ def test_partial_key_match():
 def test_dict_keys_same():
     dicts = [{'x': 42}, {'x': 23, 'y': 5}]
     assert util.dict_keys_same(dicts) == [{'y': None, 'x': 42}, {'y': 5, 'x': 23}]
+
+
+def test_get_rectangle_coordinates():
+
+    bottom_left = SkyCoord(0 * u.arcsec, 0 * u.arcsec, frame='galactic')
+    top_right = SkyCoord(10 * u.arcsec, 10 * u.arcsec, frame='galactic')
+    width = 10 * u.arcsec
+    height = 10 * u.arcsec
+    
+    with pytest.raises(ValueError):
+        util.get_rectangle_coordinates(bottom_left)
```

Comment:
Good testing practice is for each test function to test one thing, so each call to `get_rectangle_coordinates` should be in it's own test function. To prevent repeating yourself with the parameters for the calls, I suggest you make a fixture for then all:

```python
@pytest.fixture
def rectangle_args():
    ...
    return bottom_left, top_right, width, height

def test1(rectangle_args):
    bottom_left, top_right, width, height = rectangle_args
    ...
```